TopsortAnalytics/src/test/java/com/topsort/analytics/TopsortAnalyticsHttpServiceTest.kt

```diff
@@ -0,0 +1,87 @@
+package com.topsort.analytics
+
+import com.topsort.analytics.model.ClickEvent
+import com.topsort.analytics.model.Impression
+import com.topsort.analytics.model.ImpressionEvent
+import com.topsort.analytics.model.Placement
+import com.topsort.analytics.model.PurchaseEvent
+import com.topsort.analytics.model.PurchasedItem
+import com.topsort.analytics.model.Session
+import com.topsort.analytics.service.TopsortAnalyticsHttpService
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
+import org.junit.Before
+import org.junit.Test
+
+internal class TopsortAnalyticsHttpServiceTest {
+
+    private lateinit var analytics: TopsortAnalytics
+    private lateinit var service: TopsortAnalyticsHttpService
+    private lateinit var session: Session
+
+    @Before
+    fun setup() {
+        analytics = Analytics
+        service = TopsortAnalyticsHttpService
```

Comment:
Would be great to mock a server inc responses; i don't think we wanna fire to a prod/test environment here.